themes/wim/js/wim.js

```diff
@@ -34,10 +34,15 @@
       var $topTasksMenu = $('section.menu-top-tasks'),
         openDefaultClass = 'open-default';
 
+      if (isMobile()) {
+        $('li.first.dropdown', $topTasksMenu).addClass('open');
```

Comment:
@Crazylord See how the components works on www.wimgemeenten.nl and make sure to copy that behaviour. On mobile it's open by default. Are you sure this the way you detect a mobile device here works similar to the breakpoint we use in the MQ mixin for breakpoints?